docs/src/reference/the-traversal.asciidoc

```diff
@@ -1376,6 +1376,24 @@ metrics = t.getSideEffects().get('metrics')
 
 For traversal compilation information, please see <<explain-step,`explain()`>>-step.
 
+[[project-step]]
+Project Step
+~~~~~~~~~~~~
+
+The `project()`-step (*map*) projects labeled attributes of the current object into a `Map<String,Object>`. It is similar
+to <<select-step,`select()`>>-step, save that instead of retrieving and modulating historic traverser state, it modulates
+the current state of the traverser.
+
+[gremlin-groovy,modern]
+----
+g.V().out('created').
+           project('a",'b').
```

Comment:
I just fixed. I built the docs, then reformatted the text and I caused that. The docs are rebuilding right now... then I will push the typo fix.